sounds/management/commands/analysis_orchestrator.py

```diff
@@ -0,0 +1,96 @@
+# Needs to be updated everytime a new analyzer or analyzer version is added
+ANALYZERS_DICT = {
+    "fs-essentia:1": "fs-essentia-extractor:20210525_9a3bd10",
+    "audio-commons:1": "ac-extractor:20210525_9a3bd10",
+    "audioset-vggish:1": "audioset-vggish-extractor:20210615_f44030e"
+}
+
+
+class Command(BaseCommand):
+
+    help = """Checks if there are sounds that have not been analyzed by the analyzers defined in the Analysis 
+    Configuration File (or that are analyzed with older versions) and send jobs to the analysis workers if needed"""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--analysis_config_file',
+            action='store',
+            required=True,
+            help='Absolute path to the analysis configuration file')
+        parser.add_argument(
+            '--status',
+            action="store_true",
+            help='Flag to print the main status of the analysis. It does not trigger any analysis.')
+
+    def handle(self, *args, **options):
+        # Read config json
+        with open(options['analysis_config_file']) as json_file:
+            config_file = json.load(json_file)
+        
+        # Print information about the already analyzed sounds if the status flag exists
+        if options['status']:
+            n_analyzed = len(SoundAnalysis.objects.all())
+            n_sounds = len(Sound.objects.all())
+            n_analyzers = len(config_file['analyzer:version'])
+            total2analyze = n_sounds * n_analyzers
+            console_logger.info("{0} analysis performed. In total, there can be {1} ({2} sounds x {3} analyzers in config file).".format(n_analyzed, total2analyze, n_sounds, n_analyzers))
+            
+            # Count number of analysis to be performed
+            n_analysis = 0
+            for a in config_file['analyzer:version']:
+                # Check all sounds available
+                for s in Sound.objects.all():
+                    _, version = a.split(":")
+                    if not SoundAnalysis.objects.filter(sound=s, analyzer=ANALYZERS_DICT[a], analyzer_version=version).exists():
+                        n_analysis += 1
+            console_logger.info("{0} analysis to be performed.".format(n_analysis))
+
+            # List all saved analysis
+            console_logger.info("List of analyzed sounds:")
+            for a in SoundAnalysis.objects.all():
+                console_logger.info(json.dumps({'sound_id': a.sound.id, 'SoundAnalysis_id': a.id,
+                                                'analyzer': a.analyzer, 'analyzer_version': a.analyzer_version
+                                                }))
+            
+        else:
+            console_logger.info("Analysis configuration file: {0}".format(config_file))
+            for a in config_file['analyzer:version']:
+                # Check all sounds available
+                for s in Sound.objects.all():
```

Comment:
This will not be simply "all sounds", it will be only the sounds selected with the flags that I mentioned before. So this could be sounds which are "missing" (have no `SoundAnalysis` object for that analyzer/version), sounds that failed or skipped, etc...